Automattic/jetpack
# Sync Package: Moves the Users and Themes sync module

We are trying to move away from class mapped packages, and unspool all of the dependencies within a package.

So let's do this!




Part of #12712

#### Changes proposed in this Pull Request:

This PR should have no functional changes. The Users and Themes sync modules should function exactly the same as before.


#### Is this a new feature or does it add/remove features to an existing part of Jetpack?

This modifies an existing part of Jetpack.

#### Testing instructions:

* Try this branch out on a Jurassic.ninja site or on your docker testing site.
* Connect the site and activate the recommended features.
* Point your testing site to your .com sandbox to observe sync actions in real time.
* Try a full sync and ensure there are no php errors in your site's error logs
* Try adding, deleting, update some users on your site. Ensure you see the expected sync actions in the pipeline.

#### Proposed changelog entry for your changes:

* None
